File: fidouafclient/src/main/java/io/hanko/fidouafclient/asm/msgs/request/AuthenticateIn.java
package io.hanko.fidouafclient.asm.msgs.request;

import io.hanko.fidouafclient.client.msg.Transaction;

public class AuthenticateIn {
    public String appID;
    public String[] keyIDs;
    public String finalChallenge;
    public Transaction transaction;
}

File: fidouafclient/src/main/java/io/hanko/fidouafclient/asm/msgs/request/AuthenticateIn.kt
package io.hanko.fidouafclient.asm.msgs.request

import com.squareup.moshi.JsonClass
import io.hanko.fidouafclient.client.msg.Transaction

@JsonClass(generateAdapter = true)
data class AuthenticateIn (
    val appID: String,
    val keyIDs: List<String>,
    val finalChallenge: String,
    val transaction: Transaction?
)
